indico/queries/teach_tasks.py

```diff
@@ -0,0 +1,227 @@
+import json
+from typing import List, Any
+
+import pandas as pd
+
+from indico.queries.datasets import CreateDataset, GetDataset
+
+from indico.client.request import (
+    GraphQLRequest,
+    RequestChain,
+)
+
+from indico.types.questionnaire import Questionnaire, Example
+from indico.errors import IndicoNotFound
+
+
+class DataFileName(GraphQLRequest):
+    query = """
```

Comment:
you should add docstrings to each of the new methods in the same format as our existing prebuilt queries. this will be automatically rendered by sphinx doc generator